markdown/blockprocessors.py

```diff
@@ -314,6 +308,22 @@ class OListProcessor(BlockProcessor):
     # List of allowed sibling tags.
     SIBLING_TAGS = ['ol', 'ul']
 
+    def __init__(self, parser):
+        BlockProcessor.__init__(self, parser)
```

Comment:
Looks like I missed updating a class to new style classes in 794bffb4f1. As the [release notes][1] stated (for 2.3 -- that was a while ago):

> All classes are now “new-style” classes. In other words, all classes subclass from ‘object’.

However, I can clearly see that [Blockprocessor][2] does not. Looks like this whole file needs to be updated to be consistent with the rest of the library. If want to do that, great. If not, I'll take care of it myself.

[1]: https://pythonhosted.org/Markdown/release-2.3.html#backwards-incompatible-changes
[2]: https://github.com/waylan/Python-Markdown/blob/e594213ab689847ee7d9654817b8fa80fafb0fb7/markdown/blockprocessors.py#L41